### PR TITLE
Add Settings/Help as per other Foundation apps

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
- Add svelte-headless to give us an accessible menu control. It's
  not viable to make it accessible some supporting code so I've gone
  with a low impact/customisable option.
- Introduce an icon library that has access to relevant icons (we largely
  use Remix though there are exceptions).
- Add support for accessing the app version in the UI
- Disable translation tests that require us to mainain stub Danish
  strings. Remove those strings. svelte-i18n falls back to English.
  
  Closes https://github.com/microbit-foundation/ml-trainer/issues/48

### CODE REVIEW

File: src/components/control-bar/control-bar-items/HelpMenu.svelte [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<AboutDialog
+  isOpen={isAboutDialogOpen}
+  onClose={() => {
+    isAboutDialogOpen = false;
+  }} />
+<div class="relative inline-block">
+  <button
+    use:menu.button
+    class="inline-flex rounded-full text-xl p-2"
+    aria-label="Settings">
```
Comment (microbit-grace) [inline src/components/control-bar/control-bar-items/HelpMenu.svelte:26]: Help menu and translated

File: src/components/control-bar/control-bar-items/SettingsMenu.svelte [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<LanguageDialog
+  isOpen={isLanguageDialogOpen}
+  onClose={() => {
+    isLanguageDialogOpen = false;
+  }} />
+<div class="relative inline-block">
+  <button
+    use:menu.button
+    class="inline-flex rounded-full text-xl p-2"
+    aria-label="Settings">
```
Comment (microbit-grace) [inline src/components/control-bar/control-bar-items/SettingsMenu.svelte:22]: translate

File: src/components/MenuTransition.svelte [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import Transition from 'svelte-transition';
+  export let show: boolean;
+</script>
+
+<Transition
```
Comment (microbit-matt-hillsdon) [inline src/components/MenuTransition.svelte:12]: This is exactly as per svelte-headless's docs for the menu. I tried and failed to customise it to look a bit more like the other Foundation apps. Seems good enough to me!

File: src/components/control-bar/control-bar-items/AboutDialog.svelte [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+</script>
+
+<StandardDialog {isOpen} {onClose}>
+  <div class="flex flex-col gap-5 w-150 text-black">
+    <div class="flex flex-col gap-8">
+      <div class="grid grid-cols-2 gap-20 items-center">
+        <img src={microbitLogo} alt="micro:bit" />
+        <img src={aarhusLogo} alt="Aarhus" />
+      </div>
+      <p class="text-center">
```
Comment (microbit-matt-hillsdon) [inline src/components/control-bar/control-bar-items/AboutDialog.svelte:27]: We'd like this to contain a link but need to look at svelte-i18n's support for rendering links in translated text which I've decided to come back to.

File: src/components/control-bar/control-bar-items/AboutDialog.svelte [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+  export let onClose: () => void;
+  export let isOpen: boolean;
+</script>
+
+<StandardDialog {isOpen} {onClose}>
+  <div class="flex flex-col gap-5 w-150 text-black">
+    <div class="flex flex-col gap-8">
+      <div class="grid grid-cols-2 gap-20 items-center">
+        <img src={microbitLogo} alt="micro:bit" />
```
Comment (microbit-matt-hillsdon) [inline src/components/control-bar/control-bar-items/AboutDialog.svelte:24]: Use of logos and form of words here all subject to discussion. But we'll merge as a starting point.

**Review by microbit-grace — APPROVED** [review]
Looks good to me

**Pull request merged**